src/libs/actions/IOU.ts

```diff
@@ -6679,6 +6679,7 @@ function putOnHold(transactionID: string, comment: string, reportID: string) {
                 comment: {
                     hold: null,
                 },
+                errors: ErrorUtils.getMicroSecondOnyxError('report.genericAddCommentFailureMessage'),
```

Comment:
The mentioned error genericAddCommentFailureMessage -> `Unexpected error while posting the comment, please try again later.` does not sound correct here. Can you please use a generic hold related error message along with Spanish translation?

Reply:
Preferably in iou->error makes more sense to me.